doc/en/backwards-compatibility.rst

```diff
@@ -15,91 +15,6 @@ We will only remove deprecated functionality in major releases (e.g. if we depre
 Deprecation Roadmap
 -------------------
 
-This page lists deprecated features and when we plan to remove them. It is important to list the feature, the version where it got deprecated and the version we plan to remove it.
+We track deprecation and removal of features using milestones and the `deprecation <https://github.com/pytest-dev/pytest/labels/type%3A%20deprecation>`_ and `removal <https://github.com/pytest-dev/pytest/labels/type%3A%20removal>`_ labels on GitHub.
```

Comment:
should we perhaps link to the label page which shows both open and closed? (for example https://github.com/pytest-dev/pytest/issues?q=label%3A%22type%3A+deprecation%22)

Reply:
Good idea, done!